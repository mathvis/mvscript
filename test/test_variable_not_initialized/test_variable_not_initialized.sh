#!/bin/bash
helix test_variable_not_initialized.mvs test_variable_not_initialized.out --vsplit
