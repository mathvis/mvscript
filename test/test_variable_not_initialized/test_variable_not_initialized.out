error: Variable "a" was not initialized.
 --> test/test_variable_not_initialized/test_variable_not_initialized.mvs:1:1
  |
1 | a = 0;
  | ^
  |
hint: Consider using the let keyword.
